specs/palindrome_check_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative '../lib/palindrome_check'
+
+describe "palindrome" do
+  describe "basic tests" do
+    describe "test 1" do
```

Comment:
Same as the other repos, we should probably remove the `describe` blocks that are immediate parents of individual tests.